conf/timeseries-filodb-server.conf

```diff
@@ -5,6 +5,13 @@ filodb {
     port = 9042
     partition-list-num-groups = 1
   }
+  spread {
+    default = 1
+    override {
+      App-0 = 2
+      App-5 = 2
```

Comment:
With potential multiple key support I think a better config structure would be to have the partition key column/label name be in the config.  This also makes things much clearer, instead of an implicit first non-metric shard key.  So like this:

```
   override {
       _ns {
          App-0 = 2
       }
    }
```

Reply:
Multiple worskpaces can also have same application/namespace name?
So should it be something like
```
override {
	ws1 {
		_ns {
			App - 0 = 2
		}
	},
	ws2 {
		_ns {
			App - 0 = 3
		}
	}
}
```

Reply:
I have changed config for spread-override as discussed